src/components/PuzzleBoard/puzzleBoard.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+class PuzzleBoard extends React.Component {
+  static defaultProps = {
```

Comment:
I think you mean to use `static propTypes = {};`, which declares the name and type of each prop that will be passed in, for example:

`puzzle: PropTypes.array.isRequired`

You'll need to install and import the prop-types library. See https://github.com/facebook/prop-types.

`defaultProps` does something slightly different.

Reply:
Ohh. Ok. Does `defaultProps` give the props default values if nothing is passed in from the parent? Do I want both? Do I need `propTypes`?

Reply:
You're correct about what `defaultProps` does. Whether you need it or not depends on the behavior of your component. Is the prop in question required to be passed in? Or is there a case where it can be not passed in and PuzzleBoard will furnish a default value. 

Adding propTypes is a good thing to do - it provides documentation about what names and types your component needs in order to function. In dev environments, react emits a warning in the console if you pass in a prop of the wrong type.